sphinx/developers/conversion.rst

```diff
@@ -7,8 +7,8 @@ using Bio-Formats version 4.2 and later.
 Working example code is provided in
 :download:`FileConvert.java <examples/FileConvert.java>` - code from that
 class is referenced here in part. You will need to have
-**bioformats_package.jar** in your Java CLASSPATH in order to compile
-FileConvert.java.
+:file:`bioformats_package.jar` in your Java :envvar:`CLASSPATH` in order to compile
```

Comment:
To keep this PR minimal and focus on the infrastructure changes required to publish on readthedocs, could we extract these changes to a separate PR?

Reply:
yes. I made more changes initially required for RTD but I had to roll them back.

Reply:
done